docker-compose.yml

```diff
@@ -8,6 +8,15 @@ services:
     volumes:
       - /var/lib/postgresql
     command: echo true
+    
+  nginx:
+    build: .
+    ports:
+        - "80:80"
```

Comment:
@shivamMg I understand now. Thank you. 
@amitkumarj441 
But still, using '.' as the build file won't do anything. It will just re-run the web service (which is pointless). In this case, we will have to create a new Dockerfile for nginx and use it in the nginx service build. See [this article](https://realpython.com/blog/python/dockerizing-flask-with-compose-and-machine-from-localhost-to-the-cloud/) for an example.